.changeset/swift-baboons-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+use `fromConfig` for all scaffolder helpers, and use the url protocol for app-config location entries
```

Comment:
Let's have a diff or instructions for how to update existing apps, so that they end up in the changelog